src/controllers/graphql/resolvers/post/postResolver.js

```diff
@@ -1,28 +1,110 @@
-const { AuthenticationError } = require('apollo-server');
 const db = require('../../connectors');
-
 const Post = db.sequelize.models.post;
-const ReplyService = require('../../../../services/replyService');
 const PostService = require('../../../../services/postService');
 const { FETCH_TOP_X_POSTS } = require('../../../../utils/constants');
+const {
+  AuthenticationError,
+  gql,
+  makeExecutableSchema,
+} = require('apollo-server');
+
+const typeDefs = gql`
```

Comment:
Each resolver should have `typeDefs`

Reply:
typeDefs are located in the graphql/types folders.

Reply:
got it